services/restund/Makefile

```diff
@@ -18,6 +18,8 @@ WIRE_VERSION    := 6
 EXTRA_MODULES   := zrest drain
 
 DOCKER          ?= true
+# FIXME: use quay.io/wire
```

Comment:
☝️

Reply:
The other Makefiles use quay.io/wire . This Makefile is an outlier. I wanted to make it clear something was strange, while making a minor improvement by allowing us to change the username, and test.